microsoft-edge/web-platform/web-platform.md

```diff
@@ -10,7 +10,7 @@ ms.date: 11/28/2022
 ---
 # Web platform overview
```

Comment:
@captainbrosset @Reezaali - change page title from "Web platform overview" to be similar to new toc bucket name? that's:
Development tips for Microsoft Edge
https://learn.microsoft.com/en-us/microsoft-edge/web-platform/web-platform

Reply:
Yes I do think we should change the page title too.

Reply:
retitled globally same as bucket name